monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -186,12 +186,21 @@ class AppComponent extends AuthComponent {
     }
   }
 
+  getSideNavHeader() {
+    if(this.state.islandMode === 'ransomware'){
+      return "Ransomware"
+    } else {
+      return "Custom"
+    }
```

Comment:
Can't we just return the island mode and capitalize the first letter in the CSS ?

Reply:
Until the back end returns `credential_stealing` for example. The title should probably come from back end as well, but we'll invest into that infrastructure during the scenarios feature